src/GitHub.VisualStudio/Commands/LinkCommandBase.cs

```diff
@@ -46,7 +50,7 @@ protected ILocalRepositoryModel GetRepositoryByPath(string path)
                 if (!string.IsNullOrEmpty(path))
                 {
                     var repo = ServiceProvider.TryGetService<IGitService>().GetRepository(path);
-                    return new LocalRepositoryModel(repo.Info.WorkingDirectory.TrimEnd('\\'), GitService.GitServiceHelper);
+                    return GitService.GitServiceHelper.CreateLocalRepositoryModel(repo.Info.WorkingDirectory.TrimEnd('\\'));
```

Comment:
Is `TrimEnd` needed here? If so, should that logic be put into `CreateLocalRepositoryModel`?